service/service_test.go

```diff
@@ -227,33 +232,33 @@ func TestClose(t *testing.T) {
 			So(serverMock.ShutdownCalls(), ShouldHaveLength, 1)
 		})
 		/*
-		   TODO - figure out if this has been added to the boilerplate code or been removed from the csv exporter,
-		   such that, do we need to make this work here
-
-		   		Convey("If service times out while shutting down, the Close operation fails with the expected error", func() {
-		   			cfg.GracefulShutdownTimeout = 1 * time.Millisecond
-		   			timeoutServerMock := &mock.HTTPServerMock{
-		   				ListenAndServeFunc: func() error { return nil },
-		   				ShutdownFunc: func(ctx context.Context) error {
-		   					time.Sleep(2 * time.Millisecond)
-		   					return nil
-		   				},
-		   			}
-
-		   			svcList := service.NewServiceList(nil)
-		   			svcList.HealthCheck = true
-		   			svc := service.Service{
-		   				Config:      cfg,
-		   				ServiceList: svcList,
-		   				Server:      timeoutServerMock,
-		   				HealthCheck: hcMock,
-		   			}
-
-		   			err = svc.Close(context.Background())
-		   			So(err, ShouldNotBeNil)
-		   			So(err.Error(), ShouldResemble, "context deadline exceeded")
-		   			So(len(hcMock.StopCalls()), ShouldEqual, 1)
-		   			So(len(timeoutServerMock.ShutdownCalls()), ShouldEqual, 1)
-		   		})*/
+			   TODO - figure out if this has been added to the boilerplate code or been removed from the csv exporter,
+			   such that, do we need to make this work here
+
+					Convey("If service times out while shutting down, the Close operation fails with the expected error", func() {
+						cfg.GracefulShutdownTimeout = 1 * time.Millisecond
+						timeoutServerMock := &mock.HTTPServerMock{
+							ListenAndServeFunc: func() error { return nil },
+							ShutdownFunc: func(ctx context.Context) error {
+								time.Sleep(2 * time.Millisecond)
+								return nil
+							},
+						}
+
+						svcList := service.NewServiceList(nil)
+						svcList.HealthCheck = true
+						svc := service.Service{
+							Config:      cfg,
+							ServiceList: svcList,
+							Server:      timeoutServerMock,
+							HealthCheck: hcMock,
+						}
+
+						err = svc.Close(context.Background())
+						So(err, ShouldNotBeNil)
+						So(err.Error(), ShouldResemble, "context deadline exceeded")
+						So(len(hcMock.StopCalls()), ShouldEqual, 1)
+						So(len(timeoutServerMock.ShutdownCalls()), ShouldEqual, 1)
+					})*/
```

Comment:
Please remove the added tab/space.
Do we need it?

Reply:
Gofmt does get a bit confused on my machine tabbing commented code.
I would be happy to remove the commented block, as that is cleaner. Would you prefer that?

Reply:
Yes I rather remove commented code.

Reply:
Commented out code removed.